sdk/core/azure-core-amqp/pom.xml

```diff
@@ -134,16 +134,16 @@
         </configuration>
       </plugin>
 
-      <plugin>
-        <groupId>org.apache.maven.plugins</groupId>
-        <artifactId>maven-compiler-plugin</artifactId>
-        <version>3.8.1</version> <!-- {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} -->
-        <configuration>
-          <compilerArgs>
-            <arg>-Xlint:deprecation</arg>
-          </compilerArgs>
-        </configuration>
-      </plugin>
+<!--      <plugin>-->
+<!--        <groupId>org.apache.maven.plugins</groupId>-->
+<!--        <artifactId>maven-compiler-plugin</artifactId>-->
+<!--        <version>3.8.1</version> &lt;!&ndash; {x-version-update;org.apache.maven.plugins:maven-compiler-plugin;external_dependency} &ndash;&gt;-->
+<!--        <configuration>-->
+<!--          <compilerArgs>-->
+<!--            <arg>-Xlint:deprecation</arg>-->
+<!--          </compilerArgs>-->
+<!--        </configuration>-->
+<!--      </plugin>-->
```

Comment:
Can this be deleted?

Reply:
I want to add this back in the future to prevent Azure Core AMQP from using deprecated APIs (other than those suppressed) but the new version of Reactor deprecates Processors which will require a sizeable conversion